src/conversion/azure/azure_converter.ts

```diff
@@ -1,15 +1,16 @@
-import {ForwardRuleSpec, NodeSpec, SymbolStore} from '../../graph';
+import {ForwardRuleSpec, NodeSpec, SymbolStore} from '../..';
```

Comment:
I'd like to discuss the pros/cons of this change. In general, I have been trying to be more specific about import paths when importing from the same package. I'd like to use index.ts to document the public API surface for users of the package.